Makefile

```diff
@@ -1,14 +1,24 @@
-.PHONY: install \
-        clean
+.PHONY: 
```

Comment:
can we have some small description what's happening here without the need to go into different scripts ?

Reply:
fixed